leetcode/string.md

```diff
@@ -0,0 +1,22 @@
+# Intervals
```

Comment:
Your pull request LEETCODE-1022 Added reverse-words-in-a-string-iii/n

Your commit: LEETCODE-1022 Added reverse-words-in-a-string-iii

Your commit: LEETCODE 1022 Refactored reverse-words-in-a-string
You need to write "-" between prefix of your message and group number
Your commit: LEETCODE-1022 Refactored reverse-words-in-a-string-iii

Your commit: Merge remote-tracking branch 'origin/reverse-words-in-a-string-iii' into reverse-words-in-a-string-iii

# Conflicts:
#	leetcode/string.md
You need to write "-" between prefix of your message and group number
Your commit: LEETCODE-1022 Refactored reverse-words-in-a-string-iii

Your commit: LEETCODE-1022 Refactored reverse-words-in-a-string-iii